requirements-dev.txt

```diff
@@ -7,7 +7,7 @@
 flake8==5.0.4
 flake8-bugbear==22.9.23
 flake8-unused-arguments==0.0.11
-coverage==5.5
+coverage[toml]==5.5
```

Comment:
I've not seen this syntax before.. what does it mean?

Reply:
This pip `extras` syntax is broadly used...  See example 7: https://pip.pypa.io/en/stable/cli/pip_install/#examples

https://coverage.readthedocs.io/en/latest/config.html#configuration-reference
> Coverage.py will read from `pyproject.toml` if TOML support is available, either because you are running on Python 3.11 or later, or because you installed with the `toml` extra (`pip install coverage[toml]`).